ocs_ci/ocs/scale_lib.py

```diff
@@ -998,3 +1000,251 @@ def get_max_pvc_count():
+                        "initialDelaySeconds": 3,
+                        "timeoutSeconds": 10,
+                    }
+                    pod_data["spec"]["template"]["spec"]["containers"][0][
+                        "livenessProbe"
+                    ] = liveness
+                    pod_data["spec"]["replicas"] = 1
+                    pod_data["spec"]["triggers"] = [{"type": "ConfigChange"}]
+                    pod_data["spec"]["paused"] = False
+                    del pod_data["spec"]["template"]["spec"]["containers"][0]["command"]
+                    del pod_data["spec"]["template"]["spec"]["containers"][0]["stdin"]
+                    del pod_data["spec"]["template"]["spec"]["containers"][0]["tty"]
+                    flag = 0
+                elif flag:
+                    # Update pod yaml with liveness probe and IO
+                    pod_data["spec"]["containers"][0]["args"] = [
+                        "/bin/sh",
+                        "-c",
+                        f"fio --name=fio-rand-readwrite --filename={liveness_check_path}/abc "
+                        f"--readwrite=randrw --bs=4K --direct=1 --numjobs=1 --time_based=1 "
+                        f"--runtime=3600000 --size=512M --iodepth=4 --fsync_on_close=1 "
+                        f"--rwmixread=25 --ioengine=libaio --rate=2k",
+                    ]
+                    liveness = {
+                        "exec": {"command": ["sh", "-ec", "df /mnt"]},
+                        "initialDelaySeconds": 3,
+                        "timeoutSeconds": 10,
+                    }
+                    pod_data["spec"]["containers"][0]["livenessProbe"] = liveness
+                    del pod_data["spec"]["containers"][0]["command"]
+                    del pod_data["spec"]["containers"][0]["stdin"]
+                    del pod_data["spec"]["containers"][0]["tty"]
+                    flag = 0
+
+                if node_selector:
+                    pod_data["spec"]["template"]["metadata"]["labels"] = node_selector
+
+            temp_list.clear()
+            pods_list.append(pod_data)
+
+    return pods_list
+
+
+def get_pod_creation_time_in_kube_job(kube_job_obj, namespace, no_of_pod):
+    """
+    Function to get pod creation time of pods created using kube_job
+    Note: Function doesn't support DeploymentConig pods
+
+    Args:
+        kube_job_obj (obj): Kube Job Object
+        namespace (str): Namespace of PVC's created
+        no_of_pod (int): POD count
+
+    Return:
+        pod_dict (dict): Dictionary of pod_name with creation time.
+
+    """
+    job_get_output = kube_job_obj.get(namespace=namespace)
+    pod_dict = dict()
+    for i in range(no_of_pod):
+        started_at_str = job_get_output["items"][i]["status"]["containerStatuses"][0][
+            "state"
+        ]["running"]["startedAt"]
+        start_time_str = job_get_output["items"][i]["status"]["startTime"]
+        started_at = re.search(r"(\d\d):(\d\d):(\d\d)", started_at_str)
```

Comment:
I think this could be moved out to utils since we have few places where we use this logic, can be done in next PR.

Reply:
Sure @vasukulkarni we can enhance it